webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js

```diff
@@ -112,6 +115,13 @@ export const bulkResumeRequest = resumeTasks => {
   return API.post(url);
 };
 
+const bulkCancelRequest = cancelTasks => {
+  const ids = cancelTasks.map(task => task.id);
+  const url = new URI('/foreman_tasks/api/tasks/bulk_cancel');
+  url.setSearch('task_ids[]', ids);
+  return API.post(url);
```

Comment:
can you pass those parameters as params in the `API.post()`, instead of passing them in the url?

Reply:
I can't, the foreman wrapper doesn't send it:
``` JS  
post(url, data = {}, headers = {}) {
    return axios.post(foremanUrl(url), data, {
      headers,
    });
  },
```

Reply:
the `data` is the params.. you can send something like: `API.post(url, { task_ids: ids } )`

Reply:
Cool, didn't know ruby reads them both as params, changed!